invenio_audit_logs/services/config.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2025 CERN.
+#
+# Invenio-Audit-Logs is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Audit Logs Service Config."""
+
+from invenio_i18n import lazy_gettext as _
+from invenio_indexer.api import RecordIndexer
+from invenio_records_resources.services import pagination_links
+from invenio_records_resources.services.base import ServiceConfig
+from invenio_records_resources.services.base.config import ConfiguratorMixin, FromConfig
+from invenio_records_resources.services.base.links import Link
+from invenio_records_resources.services.records.config import (
+    SearchOptions as SearchOptionsBase,
+)
+from invenio_records_resources.services.records.facets import TermsFacet
+from invenio_records_resources.services.records.params import (
+    FacetsParam,
+    PaginationParam,
+    QueryStrParam,
+    SortParam,
+)
+from invenio_records_resources.services.records.queryparser import QueryParser
+from sqlalchemy import asc, desc
+
+from ..records import AuditLogEvent
+from . import results
+from .permissions import AuditLogPermissionPolicy
+from .schema import AuditLogSchema
+
+
+class AuditLogSearchOptions(SearchOptionsBase):
+    """Audit log search options."""
+
+    sort_default = "newest"
+    sort_default_no_query = "bestmatch"
```

Comment:
```suggestion
    sort_default = "newest"
    sort_default_no_query = "newest"
```
if there is no query string, we probably want to see the latest logs, no?